02/app.js

```diff
@@ -0,0 +1,15 @@
+const contentEl = document.querySelector('div');
+console.log(contentEl);
+
+const onDiv = function() {
+    alert('On Div');
+}
+
+const outsideDiv = function () {
+    alert('Outside Div');
+}
+
+if(contentEl) {
+    contentEl.addEventListener('mouseenter', onDiv);
+    contentEl.addEventListener('mouseleave', outsideDiv);
+}
```

Comment:
To zadanie jest ok, pętli nie trzeba robić.

PS. Spróbuj dodać ten event DOMContentLoaded, aby zobaczyć jak to działa :)